src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -928,6 +929,7 @@ const Authenticate = (() => {
         }
         
         const onfido_token = await getOnfidoServiceToken();
+        const is_onfido_pob_error = onfido_token === 'MissingPersonalDetails';
```

Comment:
can we rename the variable? it's not POB error, it's a generic personal details error code